cookbook/how_to_install_extension_from_command_line.rst

```diff
@@ -82,7 +82,7 @@ Finally, make sure to properly clean the cache:
 Querying Extension Information
 ------------------------------
 
-There are two commands that help you to get an overview about the extension.
+There are two commands that help you get an overview about the extension.
```

Comment:
Could be either way :) http://dictionary.cambridge.org/grammar/british-grammar/help-somebody-to-do